src/connector/src/sink/encoder/json.rs

```diff
@@ -628,11 +639,12 @@ mod tests {
         assert_eq!(date_value, json!(719163));
 
         let from_epoch_config = JsonEncoderConfig {
-            time_handling_mode: TimeHandlingMode::Milli,
+            time_handling_mode: TimeHandlingMode::String,
```

Comment:
Irrelevant change?

Reply:
Ah some merge issue. But this test is for `date_handling_mode` so`time_handling_mode` won't matter.